README.md

```diff
@@ -1,16 +1,47 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
-_Please be aware that `skopeo` is still a work in progress and it currently supports only registry API V2_
+`skopeo` is a command line utility that performs various operations on container images and image repositories.  Skopeo works with API V2 registries such as Docker Registries, the Atomic Registry, Private Registries, local directories and local OCI-layout directories.  Skopeo does not require a daemon to be running to perform these operations which consist of:
 
-`skopeo` is a command line utility for various operations on container images and image repositories.
+ * Inspecting an image showing it properties including its layers.
+ * Copying an image from and to various storage mechanisms.
+ * Delete an image from an image repository.
+ * When required by the repository, skopeo can pass the appropriate credentials and certificates for authentication.
+
+ Skopeo operates on the following image and repository types:
+
+ * atomic:hostname/namespace/stream:tag
```

Comment:
The `atomic:` transport is more or less deprecated nowadays, perhaps we should not mention it here.

Reply:
I agree remove it, and we should remove it from the man page as well.

Reply:
I've removed from the manpages and from here.